lib/src/migrators/division.dart

```diff
@@ -295,6 +303,27 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
     }
   }
 
+  /// Given a division operation [node], patches it to use multiplication
+  /// instead if the reciprocal of the divisor can be accurately represented as
+  /// a decimal.
+  ///
+  /// Returns true if patched and false otherwise.
+  bool _tryMultiplication(BinaryOperationExpression node) {
+    if (!useMultiplication) return false;
+    if (node.right is! NumberExpression) return false;
+    var divisor = node.right as NumberExpression;
```

Comment:
```suggestion
    var divisor = node.right;
    if (divisor is! NumberExpression) return false;
```

This will let you avoid the `as`.